.github/workflows/run_linter_yaml.yaml

```diff
@@ -0,0 +1,32 @@
+# Author: James Garriss
+# Purpose: Run the linter against YAML files
+
+name: Lint Check for YAML
+
+# Run whenever YAML files are checked in
+on:
+  push:
+    paths:
+      - '**.yml'
+      - '**.yaml'
+  workflow_dispatch:
+
+jobs:
+  yaml-lint-check:
+    name: MegaLint YAML Check
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      - name: Setup Config File
+        run: cp Testing/Linting/MegaLinter/.mega-linter-yaml.yml .mega-linter.yml
+        if: always()
+      - name: Setup Lint File
+        run: cp Testing/Linting/MegaLinter/.yamllint.yml .yamllint.yml
+        if: always()
+      - name: Lint YAML
+        uses: oxsecurity/megalinter/flavors/ci_light@v7.6.0
+        if: always()
```

Comment:
As in previous commentm not sure we want always here.  If previous steps are not successful then this is of no use to always run.